WordPress/dashicons
# Update: admin-site-*

Before:

![screen shot 2017-07-07 at 19 43 56](https://user-images.githubusercontent.com/4389/27972007-a48c20ca-634c-11e7-90b5-1d4493efcce7.png)

This PR:

![screen shot 2017-07-07 at 19 43 42](https://user-images.githubusercontent.com/4389/27972019-ad6e3cfa-634c-11e7-9d71-b0e64c77909f.png)

### To test

* Open `icon-font/dashicons.html`
* Check the icons are correct
